eternalcore-core/src/main/java/com/eternalcode/core/feature/afk/AfkTask.java

```diff
@@ -0,0 +1,38 @@
+package com.eternalcode.core.feature.afk;
+
+import org.bukkit.Bukkit;
+import org.bukkit.Server;
+import org.bukkit.entity.Player;
+
+import java.util.UUID;
+
+public class AfkTask implements Runnable {
+
+    private final AfkService afkService;
+    private final Server server;
+
+    public AfkTask(AfkService afkService, Server server) {
+        this.afkService = afkService;
+        this.server = server;
+    }
+
+    @Override
+    public void run() {
+        this.markAllInactivePlayers();
+    }
+
+    void markAllInactivePlayers() {
```

Comment:
jest sens wydzielania tego?

Reply:
Stwierdzilismy z Rollczim że będzie to wtedy bardziej clean